templates/comments/media-css.html

```diff
@@ -91,5 +91,19 @@
         .comments.top-level-comments {
             margin-bottom: -3px;
         }
+
+        .bad-comment {
```

Comment:
You renamed this to `bad-comment-body` in the template, is this still correct here?

Reply:
Refers to https://github.com/DMOJ/site/blob/hiding_comments/templates/comments/list.html#L11